components/specification/samples/2016-01/minimum-specification.ome.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OME xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://www.openmicroscopy.org/Schemas/OME/2016-DEV0
+                             http://www.openmicroscopy.org/Schemas/OME/2016-DEV0/ome.xsd">
+  <Image ID="Image:1" Name="Name92">
+    <AcquisitionDate>2009-03-16T15:41:51.0Z</AcquisitionDate>
```

Comment:
Once we have the final version of the samples finalized and tested, we will certainly need to review the hard-coded code bits in the model/formats documentation.